README.md

```diff
@@ -291,3 +291,30 @@ module_test:
   box: centos6
   options: --module-prs qpid/12
 ```
+
+## Client Testing With Docker
+
+The docker/clients directory contains setup and configuration to register clients via subscription-manager using an activation key and start katello-agent. The following are pre-requisites:
+
+ * (docker)[https://docs.docker.com/installation/]
+ * (docker-compose)[https://docs.docker.com/compose/install/]
```

Comment:
As these are readily available (or at least F22 which I used), can you also include a cut-and-paste bit?

```
yum install -y docker docker-compose
systemctl start docker
```